src/test/java/mops/presentation/GroupControllerTest.java

```diff
@@ -75,7 +73,7 @@ void setUp() {
      */
     @Test
     public void getAllFilesOfDirectory() throws Exception {
-        SecurityContextUtil.setupSecurityContextMock("userName", "userEmail@mail.de", Set.of("studentin"));
+        setupSecurityContextMock("user", "user@mail.de", "studentin");
```

Comment:
See above for, I wouldn't add a constructor just for the test. It's best to use the code which is actually used.

Reply:
line length